_posts/2021-04-08-newsletter-7.markdown

```diff
@@ -0,0 +1,50 @@
+- A lot of the labels that are generated are the byproduct of complicated, proprietary estimation methods. In the case of Garmin, the messages are derived from [the FirstBeat Technologies Ltd. VO2 max estimation algorithm](https://www.firstbeat.com/wp-content/uploads/2015/10/white_paper_vo2_estimation.pdf), which pulls in data points from heart rate and speed. Not only might these be inaccurate, but they also are limited to the set of data that exists within a user account already.
+
+### How the messing can go wrong
+
+Here's another example of why we really need to rely on more than just running data to frame our experiences and sense of accomplishment. This past summer I was in the throes of recovering from an injury, and I was instructed to do some interval running and focus on increasing my cadence. Because I was taking a running approach that was not reflective of the way I normally run, specifically one in which my heart rate would recover in between high-cadence intervals of running, Garmin started to interpret the results as an overall increase in fitness my level and an overall spike in my V02 max, or the amount of oxygen one's body can absorb during exercise.
+
+In fact neither increase was really true, so as I slowly began to heal and return to longer intervals, the messages after my runs cut even more deeply. Just when I felt that I was making progress, Garmin was telling me that my fitness level was decreasing that that I was "overreaching." This is just one example, but there are probably many others. It's hard for our watches to account for temperature and humidity fluctuations. Our emotions can artificially influence heart rate data. We might be breaking in a new pair of shoes. We could be running with a backpack full of clothes. Variance is the result of many factors.
+
+### Learn to ignore the noise
+
+I anticipate that metrics is a general topic I will return to in the newsletter periodically. But for now, just remember that there are different kinds of running, different goals, and different occasions, and if we are going to take cues from the feedback that data and fitness apps give us, we need to learn how to look at the entire process flexibly.
+
+### Resolution for the 2020 New York City marathon
+
+This week the New York Road Runners announced that those who had registered for the 2020 NYC Marathon can now rank order their top two preferences regarding running the race in the future: run in 2021, run in 2022, run in 2023, or receive a refund. I am hoping for a spot in the 2021 race, so we will see what happens. In general though, this seems like a reasonable way to move forward and account for the lost year of racing. I am interested, in general, about how races across the U.S. will begin to handle make-ups and resolutions as we slowly emerge from a global pandemic.
```

Comment:
Wording of the last sentence of this paragraph feels off to me. "I am interested... about how..."